SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -22,14 +22,14 @@
 class ASpatialDebugger;
 class ASpatialMetricsDisplay;
 class UAbstractLBStrategy;
-class UActorGroupManager;
+class SpatialActorGroupManager;
 class UEntityPool;
 class UGlobalStateManager;
-class USnapshotManager;
+class SpatialSnapshotManager;
```

Comment:
Nit: Would be cool if we could alphabetize these. I know they were already out of order...

Reply:
Addressed